lib/rules/no-aria-unsupported-elements.js

```diff
@@ -0,0 +1,40 @@
+import AstNodeInfo from '../helpers/ast-node-info.js';
+import Rule from './_base.js';
+
+export function ERROR_MESSAGE_ARIA_UNSUPPORTED_PROPERTY(tag, name) {
+  return `The <${tag}> element does not support the use of ARIA roles, states, and properties such as ${name}`;
+}
+const RESERVED_ELEMENTS = new Set(['html', 'meta', 'script', 'style']);
```

Comment:
What about the name `NON_ARIA_ELEMENTS`? The meaning of "reserved" isn't clear to me.